internal/patroni/config.go

```diff
@@ -50,6 +50,17 @@ func clusterYAML(
 		// lifetime.
 		"scope": naming.PatroniScope(cluster),
 
+		// Configure the Patroni log path directory
+		// - https://patroni.readthedocs.io/en/latest/yaml_configuration.html#log
+		//
+		// Setting file_size to 0 means that there will be no rollover.
+		// - https://github.com/patroni/patroni/blob/v3.3.4/patroni/log.py#L431-L432
+		// - https://docs.python.org/3.11/library/logging.handlers.html#rotatingfilehandler
+		"log": map[string]any{
+			"dir":       naming.PatroniPGDataLogPath,
+			"file_size": 0,
```

Comment:
:thought_balloon: I'm conflicted about this. On one hand, this makes referencing the correct file simpler and removes the possibility of losing logs over time based on file size limits. However, this means the file will grow larger and larger over time if not managed. I think we should discuss this further before merging.

Reply:
Per discussion with team, this is now configurable by the user.